apis/generate.go

```diff
@@ -33,9 +33,6 @@ Copyright 2021 Upbound Inc.
 // Generate crossplane-runtime methodsets (resource.Claim, etc)
```

Comment:
@turkenf please remove this commit from your PR. I remember you did this to break to private dependency to unblock yourself but we should not merge it.

Reply:
It is gone after I rebased my PR.